README.md

```diff
@@ -53,6 +53,18 @@ tar -xvf kompose_linux-amd64.tar.gz --strip 1
 sudo mv kompose /usr/local/bin
 ```
 
+#### CentOS
+
+```bash
+sudo yum -y install centos-release-openshift-origin && sudo yum -y install kompose
```

Comment:
It would be great to add few words explaining what `centos-release-openshift-origin` is